code/__DEFINES/materials/basic.dm

```diff
@@ -0,0 +1,340 @@
+// Basic and tier-0 materials. These are typically of the lowest value, and the simplest/purest elements.
+
+#define MAT_hydro "h2"
+/datum/mat/hydrogen
```

Comment:
These datums are never instantiated.

I have two options, 1: Make a separate file, one with a bunch of defines, and one with a bunch of uninstantiated datum defines. This makes it so that if you want to find the ID of a datum, you have to go BACK to the defines folder and skim through it until you find the proper define. Or alternatively, skim through the defines and then have to use the search function to find out where its actually used, so that you know what it is.

The other option is this: to define them WITH the datums. But the issue there is that it needs to be defined before it will be used in the code. IE. before the game or module folders.